main.go

```diff
@@ -101,6 +102,10 @@ func init() {
 	utilruntime.Must(grafanav1beta1.AddToScheme(scheme))
 
 	utilruntime.Must(routev1.AddToScheme(scheme))
+
+	// Gateway API is optional - operator should function without it
+	// nolint:errcheck // Intentionally ignoring error - operator should work without Gateway API CRDs
+	_ = gatewayv1.Install(scheme)
```

Comment:
Could you explain why it's implemented this way? `Routes` are also optional, but we don't suppress their registration errors (line 104).